services/frontend-service/src/ui/components/EnvironmentCard/EnvironmentCard.tsx

```diff
@@ -45,7 +45,12 @@ export const EnvironmentCard: React.FC<{ environment: Environment }> = (props) =
                 {locks.length !== 0 && (
                     <div className="environment__locks">
                         {locks.map((lock) => (
-                            <EnvironmentLockDisplay env={environment.name} lockId={lock} key={lock} />
+                            <EnvironmentLockDisplay
+                                env={environment.name}
+                                lockId={lock}
+                                key={lock}
+                                bigLockIcon={true}
```

Comment:
I'm really curious as to what this looks like, maybe a screenshot would help?

Reply:
I added 2 screenshots to this PR @jaypontes